### PR TITLE
update DG's design section & UG's team name

### PR DESCRIPTION
**Pull request by kevinswk94**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -28,28 +28,30 @@ Refer to the guide <<SettingUp#, here>>.
 .Architecture Diagram
 image::ArchitectureDiagram.png[]
 
-The *_Architecture Diagram_* given above explains the high-level design of the App. Given below is a quick overview of each component.
+The *_Architecture Diagram_* given above explains the high-level design of CookBuddy.
+A quick overview of each component is give below.
```
Comment (sharadhr) [inline docs/DeveloperGuide.adoc]: *given

**Review by sharadhr — APPROVED** [review]
Minor spelling error

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (kevinswk94) [reply docs/DeveloperGuide.adoc]: thanks!

Comment (kevinswk94) [inline docs/DeveloperGuide.adoc]:
```suggestion
A quick overview of each component is given below.
```

**Comment by sharadhr** [comment]
Ready to merge :)

**Pull request merged**